lib/model/autocomplete.dart

```diff
@@ -192,10 +192,13 @@ class MentionAutocompleteView extends ChangeNotifier {
   final PerAccountStore store;
   final Narrow narrow;
 
+  MentionAutocompleteQuery? get query => _currentQuery;
   MentionAutocompleteQuery? _currentQuery;
-  set query(MentionAutocompleteQuery query) {
+  set query(MentionAutocompleteQuery? query) {
```

Comment:
When adding this, I got a diagnostic error
    `getter_not_subtype_setter_types`:
      https://dart.dev/tools/diagnostic-messages?utm_source=dartdev&utm_medium=redir&
utm_id=diagcode&utm_content=getter_not_subtype_setter_types#getter_not_subtype_setter
_types
    
    Shrug; I guess it's fine to adjust the setter param's type to the
    higher type, with code to handle the difference.

Cool. My first reaction was actually that it's perfectly reasonable for the setter to take a narrower type than the getter returns, for a use case like this one where the value starts as null or some other placeholder and you don't want to allow setting again to that… but on reading a bit of upstream discussion, following links from here:
https://groups.google.com/a/dartlang.org/g/misc/c/74VgbBxVXoM
the idea is that in the future the getter and setter might be more closely paired, as two aspects of a single member with a single type. Which seems like a fine future to aim for.

Reply:
Should trim down that URL in the commit message, though. Without the tracking it gets a lot more readable :slightly_smiling_face:  — and also avoids getting wrapped across lines in a terminal.

(The messed-up formatting above is how it copy-pasted for me, which typically I'd fix but here it serves as a nice illustration.)